integration-tests/tests/src/utils/sleep.ts

```diff
@@ -0,0 +1,23 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+declare function setTimeout(cb: () => void, ms: number): void;
+
+export function sleep(ms = 1000): Promise<void> {
```

Comment:
This was useful when debugging. I've written it a few times before, so this time I decided to just leave it in for the future.

Reply:
Yes, we all write this when debugging and remove it before committing. It's good to have - thanks.